api/src/couchdb/notebooks.ts

```diff
@@ -623,16 +624,35 @@ const csvFormatValue = (
   if (fieldType === 'faims-pos::Location') {
     if (value instanceof Object && 'geometry' in value) {
       result[fieldName] = value;
-      result[fieldName + '_latitude'] = value.geometry.coordinates[0];
-      result[fieldName + '_longitude'] = value.geometry.coordinates[1];
+      result[fieldName + '_latitude'] = value.geometry.coordinates[1];
+      result[fieldName + '_longitude'] = value.geometry.coordinates[0];
```

Comment:
Not sure on level of 'trust' in the validity of the data at this point - what if geometry object does not have coordinates field or the array index does not exist? Should this fail gracefully, throw an exception? Go to next record?

Reply:
Added some guards to this and the other geometry export.